dev/AppLifecycle/AppLifecycle.idl

```diff
@@ -53,20 +57,26 @@ namespace Microsoft.Windows.AppLifecycle
         // Windows.ApplicationModel.Activation.ActivationKind.
 
         Push = 5000,
+
+        [contract(AppLifecycleContract, 2)]
         AppNotification,
     };
 
+    [contract(AppLifecycleContract, 1)]
     runtimeclass AppActivationArguments
     {
         ExtendedActivationKind Kind { get; };
         IInspectable Data{ get; };
     };
 
+    [contract(AppLifecycleContract, 1)]
     runtimeclass AppInstance
     {
         static AppInstance GetCurrent();
         static Windows.Foundation.Collections.IVector<AppInstance> GetInstances();
         static AppInstance FindOrRegisterForKey(String key);
+
+        [contract(AppLifecycleContract, 2)]
```

Comment:
Did @dhoehna cover whether we're using major or major.minor contract versions? There's no _technical_ problem with using just major, except that we might eventually want to make breaking changes clearly called out with the "2.0" version number.

Reply:
It was agreed that there would be a MAJOR version only.